tests/bearlib/languages/documentation/documentation_extraction_testdata/data.py

```diff
@@ -9,7 +9,12 @@ def foobar_explosion(radius):
     A nice and neat way of documenting code.
     :param radius: The explosion radius.
     """
-    return 55 * radius
+    def get_55():
+        """
+        A function that returns 55.
+        """
+        return 55
+    return get_55() * radius
```

Comment:
to check the indentation, this is a nested function, so it returns 8 as indentation.